.github/workflows/production.yaml

```
@@ -11,6 +11,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Use Node.js 20
+        uses: actions/setup-node@v2
+        with:
+          node-version: '20'
```

Similar to the `production.yml`, change this `v2` to `v4`

```suggestion
      - uses: actions/checkout@v4
```

Reference: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions